proposals/ml_fit_class_support_vector.json

```diff
@@ -0,0 +1,153 @@
+{
+    "id": "ml_fit_class_support_vector",
+    "summary": "Train a support vector classification model",
+    "description": "Executes the fit of a support vector classification model based on training data. The support vector classification model aims to find a hyperplane that best separates different classes in the feature space.",
+    "categories": [
+        "machine learning"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "predictors",
+            "description": "The predictors for the classification model as a vector data cube. Aggregated to the features (vectors) of the target input variable.",
+            "schema": [
+                {
+                    "type": "object",
+                    "subtype": "datacube",
+                    "dimensions": [
+                        {
+                            "type": "geometry"
+                        },
+                        {
+                            "type": "bands"
+                        }
+                    ]
+                },
+                {
+                    "type": "object",
+                    "subtype": "datacube",
+                    "dimensions": [
+                        {
+                            "type": "geometry"
+                        },
+                        {
+                            "type": "other"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "target",
+            "description": "The training sites for the classification model as a vector data cube. This is associated with the target variable for the support vector classification model. The geometry has to be associated with a class label.",
```

Comment:
I have a hard time understanding what this description is saying about the `target` vector cube

Reply:
Redefined.